client/src/api/api.ts

```diff
@@ -2307,6 +2307,109 @@ export interface ProfileInfoDto {
      */
     'isProfileSettingsChanged': boolean;
 }
+/**
+ * 
+ * @export
+ * @interface ProfileInfoFlatDto
+ */
+export interface ProfileInfoFlatDto {
+    /**
+     * 
+     * @type {GeneralInfo}
+     * @memberof ProfileInfoFlatDto
+     */
+    'generalInfo': GeneralInfo;
```

Comment:
You duplicate fields from `GeneralInfo` interface, make sense to do this?

Reply:
Definitely not, thanks!